### ISSUE TITLE
How do you check in/scan tickets?

### ISSUE THREAD

**Issue by cycomachead** [openSUSE/osem#3250]
**I'm submitting a ..**
- [ ] Bug Report
- [ ] Feature Request
- [X] Question

I'm running an in person (OMG!) event soon. (Finally!) How do we check off people as present when checkin tickets? There's a route for ticket scanning, but I don't see any front end connected to it...or am I missing something?

Thanks for the help!

**Comment by hennevogel** [comment]
I'm not quite sure this exists still, there used to be a switch in `/admin/conferences/osemdemo/registrations` which got removed when @bear454 moved this to DataTables. See the discussion in #2189

**Comment by cycomachead** [comment]
Ah thanks, that's interesting. Though, that switch looked like it just toggled registration and didn't use the `TicketScanning` model. Seems to have been added in  https://github.com/openSUSE/osem/pull/1642 but I guess there was never a clear way to POST to that route?

I assumed this was meant to be used in conjunction with the QR codes on tickets somehow...For now, I think I'll just add a button to one of the ticket tables.

**Comment by cycomachead** [comment]
I see! 😰 
Does anyone use the QR codes? I feel like there's some good bits here we could make actually usable by not using a POST and having a GET with the ticket token

**Comment by hennevogel** [comment]
Yeah it's probably a pretty small feature. Encode a "check in" URL and the GUID of the `PhysicalTicket` into the QR code `PhysicalTicketsController`. Would allow people at the entrance to scan the `PhysicalTicket` with a QR code app on their phone.
